### ISSUE TITLE
Make simple enable/disable block in configuration.

### ISSUE THREAD

**Issue by devu** [KeenSoftwareHouse/SpaceEngineers#87]
On many multiplayer and especially modded servers there is a big need for simple mechanism to disable vanilla blocks for many reasons. Repeating entire entry and adding <public>false</public> seems to be very over-bloated way to deal with it. 

Wouldn't  entry like this:

<Definition xsi:type="MyObjectBuilder_AssemblerDefinition">
      <Id><TypeId>Assembler</TypeId><SubtypeId>LargeAssembler</SubtypeId></Id>
      <public>false</public> 
</Definition>

be enough?

or even introduce
<enabled>false</enabled> 
Since public might be handy to keep pre-build stuff on the server.

**Comment by zrisher** [comment]
Personally I'd prefer the building object checks a hook to see if it can be built so limits can be determine dynamically, a la #59. But the above would be a nice additional feature for a slight performance boost.

**Comment by Tyrsis** [comment]
This already exists: 
<Definition xsi:type="MyObjectBuilder_AssemblerDefinition" Enabled="false">

**Comment by devu** [comment]
If it does is not documented, could you tell us the secret? :)

**Comment by zrisher** [comment]
Yep just used this today with respawn ships - if you add that `Enabled` attribute (and make sure to use lowercase for its values), you can disable blocks at init. But this does not allow for a dynamic check during run-time.

**Comment by mexmer** [comment]
@zrisher Enabled node on blocks that derive from MyFunctionalBlock effectively sets block to On/Off state.

eg. if you for example have Turret with <Enabled>false</Enabled> this turret is loaded inactive (you need to switch it on in terminal gui)

Enabled attribute in block definition root node enables/disables block definition (it's easier than commenting whole block out, or deleting from definition file)

**Comment by zrisher** [comment]
Ah thanks, I was wondering about that distinction.

**Comment by mexmer** [comment]
Enable node is actually quite practical, and of course is used in savefiles ;o)
